### PR TITLE
Update GPU HF-Transformers example structure

### PR DESCRIPTION
**Pull request by plusbang**
## Description

**Preview link is https://github.com/plusbang/IPEX_LLM/tree/update-example-structure/python/llm/example/GPU/HuggingFace**

### 3. Summary of the change 

- New example structure
- Update reference links

### 4. How to test?
- [x] N/A

### CODE REVIEW

**Comment by jason-dai** [comment]
1. Change to
    ```
    HuggingFace\LLM
               \Multimodal
               \Advanced-Quantizations
               \More-Data-Types
               \Save-Load    
     ```
2.  Need to update links in https://github.com/intel-analytics/ipex-llm#verified-models

File: python/llm/example/GPU/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 This folder contains examples of running IPEX-LLM on Intel GPU:
 
 - [Applications](Applications): running LLM applications (such as autogen) on IPEX-LLM
-- [HF-Transformers-AutoModels](HF-Transformers-AutoModels): running any ***Hugging Face Transformers*** model on IPEX-LLM (using the standard AutoModel APIs)
+- [HuggingFace](HuggingFace): running any ***Hugging Face Transformers*** model on IPEX-LLM (using the standard AutoModel APIs), including language models and multimodal models.
```
Comment (jason-dai) [inline python/llm/example/GPU/README.md:6]: running ***Hugging Face*** models

File: python/llm/example/GPU/HuggingFace/README.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # Running HuggingFace `transformers` model using IPEX-LLM on Intel GPU
```
Comment (jason-dai) [inline python/llm/example/GPU/HuggingFace/README.md:1]: Running HuggingFace models using IPEX-LLM on Intel GPU

File: python/llm/example/GPU/HuggingFace/README.md [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
 
-This folder contains examples of running any HuggingFace `transformers` model on IPEX-LLM (using the standard AutoModel APIs):
+This folder contains examples of running any HuggingFace `transformers` model on IPEX-LLM:
```
Comment (jason-dai) [inline python/llm/example/GPU/HuggingFace/README.md:3]: running HuggingFace models

**Review by jason-dai — APPROVED** [review]
See comments inline; otherwise LGTM

**Comment by plusbang** [comment]
> See comments inline; otherwise LGTM



> See comments inline; otherwise LGTM

Have updated, will merge it first.

**Pull request merged**